packages/apps/plugins/plugin-thread/src/ThreadPlugin.tsx

```diff
@@ -396,13 +395,20 @@ export const ThreadPlugin = (): PluginDefinition<ThreadPluginProvides> => {
 
                 const qualifiedSubjectId = fullyQualifiedId(doc);
                 const attention = attentionPlugin?.provides.attention?.attended ?? new Set([qualifiedSubjectId]);
-                const attendableAttrs = useAttendable(qualifiedSubjectId);
+                const attendableAttrs = createAttendableAttributes(qualifiedSubjectId);
                 const space = getSpace(doc);
                 const context = space?.db.getObjectById(firstMainId(location?.active));
                 const { showResolvedThreads } = getViewState(qualifiedSubjectId);
 
-                const onCreateAnalytics = useAnalyticsCallback(space?.id, 'threads.thread-created');
```

Comment:
fwiw a lot of this logic inside this if statement could also be factored out into a component to prevent the hooks violation but this seems fine too